pkg/packagekit/updates.jsx

```diff
@@ -56,18 +56,43 @@ const PK_STATUS_STRINGS = {
     14: _("Verifying"),
 }
 
+const I_TRANS = "org.freedesktop.PackageKit.Transaction";
+
 var dbus_pk = cockpit.dbus("org.freedesktop.PackageKit", {superuser: "try", "track": true});
 var packageSummaries = {};
 
-function pkTransaction() {
+function pkWatchTransaction(transObj, signalHandlers, notifyHandler) {
+    var subscriptions = [];
+
+    for (let h in signalHandlers)
```

Comment:
Should we call this `handler`? It isn't used often.
Also the for loop might look better with braces `{ }`. I'm not entirely sure about this, since the loop only contains one statement, yet that is spread over two lines...